src/integration/legacy/setup.go

```diff
@@ -0,0 +1,97 @@
+// Copyright (c) 2020  Uber Technologies, Inc.
```

Comment:
This file is just `src/integration/harness_test.go` renamed and tweaked to run in process.